fq.h

```diff
@@ -65,12 +65,24 @@
 
 #define FQ_DEFAULT_QUEUE_TYPE "mem"
 
+#ifndef min
+#define min(a,b) (a < b ? a : b)
```

Comment:
I'd add parenthesis here as depending on a and b some weird shit could happen.

((a) < (b)) ? (a) : (b))